spec.html

```diff
@@ -20545,6 +21118,9 @@ <h1>Static Semantics: Early Errors</h1>
         <li>
           It is a Syntax Error if PrototypePropertyNameList of |ClassElementList| contains more than one occurrence of *"constructor"*.
         </li>
+        <li>
+          It is a Syntax Error if PrivateBoundIdentifiers of |ClassElementList| contains any duplicate entries, unless the name is used once for a getter and once for a setter and in no other entries, and the getter and setter are either both static or both non-static.
```

Comment:
This fixes the bug I pointed out, but I still don't like the lack of precision here - the spec doesn't say what a "getter" is, and it's not 100% clear what "the name" is or what it means for one to be "used for a getter". I'll ask other editors' opinions on this.